[2022-11-27T16:00:22.974+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:00:21.242974+00:00 [queued]>
[2022-11-27T16:00:23.003+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:00:21.242974+00:00 [queued]>
[2022-11-27T16:00:23.009+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T16:00:23.011+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-27T16:00:23.012+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T16:00:23.055+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): getDataToLocal> on 2022-11-27 16:00:21.242974+00:00
[2022-11-27T16:00:23.078+0000] {standard_task_runner.py:55} INFO - Started process 8403 to run task
[2022-11-27T16:00:23.103+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_youtube_api', 'getDataToLocal', 'manual__2022-11-27T16:00:21.242974+00:00', '--job-id', '362', '--raw', '--subdir', 'DAGS_FOLDER/dag_youtube_api.py', '--cfg-path', '/tmp/tmp0amsgbz5']
[2022-11-27T16:00:23.111+0000] {standard_task_runner.py:83} INFO - Job 362: Subtask getDataToLocal
[2022-11-27T16:00:23.349+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:00:21.242974+00:00 [running]> on host 1977dd9b5152
[2022-11-27T16:00:23.520+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_youtube_api
AIRFLOW_CTX_TASK_ID=getDataToLocal
AIRFLOW_CTX_EXECUTION_DATE=2022-11-27T16:00:21.242974+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-27T16:00:21.242974+00:00
[2022-11-27T16:00:23.856+0000] {logging_mixin.py:137} INFO - {'code': 400, 'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'errors': [{'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'domain': 'youtube.parameter', 'reason': 'invalidRegionCode', 'location': 'regionCode', 'locationType': 'parameter'}]}
[2022-11-27T16:00:23.857+0000] {logging_mixin.py:137} INFO - Writing I data to file...
[2022-11-27T16:00:24.014+0000] {logging_mixin.py:137} INFO - {'code': 400, 'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'errors': [{'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'domain': 'youtube.parameter', 'reason': 'invalidRegionCode', 'location': 'regionCode', 'locationType': 'parameter'}]}
[2022-11-27T16:00:24.017+0000] {logging_mixin.py:137} INFO - Writing D data to file...
[2022-11-27T16:00:24.019+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-27T16:00:24.041+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_youtube_api, task_id=getDataToLocal, execution_date=20221127T160021, start_date=20221127T160022, end_date=20221127T160024
[2022-11-27T16:00:24.089+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-27T16:00:24.125+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
